src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -185,9 +185,14 @@ protected override void CheckInterfaces(DiagnosticBag diagnostics)
 
                             Debug.Assert(!other.Equals(@interface, TypeCompareKind.ConsiderEverything));
 
+                            if (!other.Equals(@interface, TypeCompareKind.CLRSignatureCompareOptions | TypeCompareKind.IgnoreNullableModifiersForReferenceTypes))
```

Comment:
>TypeCompareKind.IgnoreNullableModifiersForReferenceTypes [](start = 103, length = 56)

It looks like adding IgnoreNullableModifiersForReferenceTypes option is redundant, and this feels very confusing to the reader of the code.